SimDataFormats/Associations/python/LCToCPAssociation_cfi.py

```diff
@@ -1,12 +1,8 @@
 import FWCore.ParameterSet.Config as cms
+from Validation.HGCalValidation.HGCalValidator_cfi import hgcalValidator
```

Comment:
A DataFormat package should not depend on Validation package (even on python level). If a dependence between two packages is needed, the dependence should go in the other direction.

Written that, a DataFormat package should not define plugins (e.g. EDProducers). By quick look `SimCalorimetry/HGCalAssociatorProducers` could be a reasonable place for the producers.

Reply:
@makortel thanks for the comment.
By moving the plugin definition into
`SimCalorimetry/HGCalAssociatorProducers`, can I keep the dependency on Validation package there or shall I still swap the dependency direction?

Reply:
I think also there the dependence should go by Validation depending on Simulation.